datasource-remote/src/test/java/cmm/apps/esmorga/datasource_remote/event/EventRemoteDatasourceImplTest.kt

```diff
@@ -131,4 +132,35 @@ class EventRemoteDatasourceImplTest {
         Assert.assertTrue(exception is EsmorgaException)
         Assert.assertEquals(errorCode, (exception as EsmorgaException).code)
     }
+
+    @Test
+    fun `given a working api when join event requested then the join event api is successfully called`() = runTest {
```

Comment:
this test should not be about the API being called, but about what the DS returns.